packages/test/service-load-test/src/utils.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fs from "fs";
+import { Loader } from "@fluidframework/container-loader";
+import { IFluidCodeDetails } from "@fluidframework/core-interfaces";
+import { LocalCodeLoader } from "@fluidframework/test-utils";
+import { ITestDriver, TestDriverTypes, ITelemetryBufferedLogger } from "@fluidframework/test-driver-definitions";
+import { createFluidTestDriver } from "@fluidframework/test-drivers";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import { assert, LazyPromise } from "@fluidframework/common-utils";
+import { pkgName, pkgVersion } from "./packageVersion";
+import { fluidExport, ILoadTest } from "./loadTestDataStore";
+import { ILoadTestConfig, ITestConfig } from "./testConfigFile";
+
+const packageName = `${pkgName}@${pkgVersion}`;
+
+const loggerP = new LazyPromise<ITelemetryBufferedLogger>(async ()=>{
+    if (process.env.FLUID_TEST_LOGGER_PKG_PATH) {
+        await import(process.env.FLUID_TEST_LOGGER_PKG_PATH);
+        const logger = getTestLogger();
+        assert(logger !== undefined, "Expected getTestLogger to return something");
+        return logger;
+    }else{
+        return { send: () => {}, flush: async () => {} };
+    }
+});
+
+const codeDetails: IFluidCodeDetails = {
+    package: packageName,
+    config: {},
+};
+
+const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]]);
+
+export async function createLoader(testDriver: ITestDriver) {
+    // Construct the loader
+    const loader = new Loader({
+        urlResolver: testDriver.createUrlResolver(),
+        documentServiceFactory: testDriver.createDocumentServiceFactory(),
+        codeLoader,
+        logger: await loggerP,
+    });
+    return loader;
+}
+
+export async function initialize(testDriver: ITestDriver) {
+    const loader = await createLoader(testDriver);
+    const container = await loader.createDetachedContainer(codeDetails);
+    container.on("error", (error) => {
+        console.log(error);
+        process.exit(-1);
+    });
```

Comment:
This event doesn't exist.

Reply:
didn't add it. just moving code. trying to not make changes